codespell_lib/data/dictionary.txt

```diff
@@ -3816,10 +3837,18 @@ blockhains->blockchains
 blockin->blocking
 bloddy->bloody
 blodk->block
+bloek->bloke
+bloeks->blokes
+bloekes->blokes
```

Comment:
Need to fix sorting. (this keeps biting me in the bum whenever I submit a PR too :laughing: )

Reply:
I was sure it was sorted! 😄

Reply:
Well @lurch if we could just change the order of the lines with special symbols (like `'` and `;`), we could use tools like `sort`, or VS Code extensions to sort the lines.

Reply:
https://github.com/codespell-project/codespell#sending-pull-requests

Reply:
The issue there is mainly `'` in contractions getting mixed up with `->`:

```
aircrafts->aircraft
aircrafts'->aircraft's
```

becomes

```
aircrafts'->aircraft's
aircrafts->aircraft
```

this is because `'` (apostrophe) has char code 39, and `-` (hyphen) has char code 45, so `aircrafts'` is moved before `aircrafts`. 

Do we even have sorting rules related to such cases?  I guess `aircrafts'` is supposed to be after `aircraft` because of the extra character?

Reply:
@lurch I didn't see that link you sent.  I guess I should've read that section... 😄